lib/colocated-babel-plugin.js

```diff
@@ -15,6 +19,53 @@ module.exports = function (babel) {
     name: 'ember-cli-htmlbars-colocation-template',
 
     visitor: {
+      Program(path, state) {
+        let allAddedImports = {};
+
+        state._colocationEnsureImport = (exportName, moduleName) => {
+          let addedImports = (allAddedImports[moduleName] = allAddedImports[moduleName] || {});
+
+          if (addedImports[exportName]) return addedImports[exportName];
```

Comment:
Doesn't this suffer from the same problem that https://github.com/ember-cli/babel-plugin-htmlbars-inline-precompile/pull/357 fixed?

Shouldn't this return a new identifier each time?